docs/conventions.md

```diff
@@ -6,16 +6,14 @@ We follow the [airbnb style](https://github.com/airbnb/javascript/tree/master/re
 
 ## Code style
 
-We mainly follow the [standard style](https://standardjs.com/rules) for JavaScript, although there may be smaller deviations.
-Run the linter (`yarn lint`) to show problems. Our eslint config can be found [here](https://github.com/Integreat/eslint-config-integreat).
-We use prettier to format code. Run (`yarn workspace <workspace> prettier --check .`) to show formatting problems.
+We use [prettier](https://prettier.io) to format code. Run `yarn workspace <workspace> prettier --check .` to show formatting problems or `yarn workspace <workspace> prettier --write .` to fix them.
```

Comment:
I think line 10 (Run then linter...) should not be deleted. A lot of linting is not done by prettier.